grpc/server/src/main/java/io/helidon/grpc/server/GrpcServerConfiguration.java

```diff
@@ -174,6 +177,22 @@ private Builder() {
          * @param config configuration instance
          * @return updated builder
          */
+        @ConfiguredOption(key = "name",
+                type = String.class,
+                value = DEFAULT_NAME,
+                description = "Name of the gRPC server")
+        @ConfiguredOption(key = "port",
+                type = Integer.class,
+                value = "" + DEFAULT_PORT,
+                description = "Specify the gRPC server port")
+        @ConfiguredOption(key = "native",
+                type = Boolean.class,
+                value = "false",
+                description = "Specify if native transport should be used.")
+        @ConfiguredOption(key = "workers",
+                type = String.class,
+                value = "Number of processors available to the JVM",
+                description = "Specify the count of threads in pool used to process HTTP requests.")
```

Comment:
From David: Hi Keith :slightly_smiling_face: No problem. I see. Well I think that the case in Resource class is a little bit different. The create method is basically just consuming those values and it is not setting them anywhere. Because of that, there would be no nicer way, where to put them. But in the case of GrpcServerConfiguration, those methods are there, even though not used directly. I think that in that case those annotations should be placed on them, since it corresponds with the functionality.
[8:46](https://proddev-paas-fmw.slack.com/archives/D03F1HK72SV/p1658245612730199)
I might vote for native  to stay on config method and just place others where they belong